static/app/utils/performance/quickTrace/quickTraceQuery.tsx

```diff
@@ -63,7 +69,16 @@ export default function QuickTraceQuery({children, event, ...props}: QueryProps)
               traceFullResults.error === null &&
               traceFullResults.traces !== null
             ) {
-              for (const subtrace of traceFullResults.traces) {
+              if (
```

Comment:
This is where the response the split from just {[transacations...]} to {transaction: [...], orphan_errors:[...]} is handled. Preserves functionality with or without flag.